node/config/def.go

```diff
@@ -83,20 +83,24 @@ func DefaultFullNode() *FullNode {
 			},
 		},
 		Fevm: FevmConfig{
-			EnableEthRPC:                 false,
-			EthTxHashMappingLifetimeDays: 0,
-			EthTraceFilterMaxResults:     500,
-			EthBlkCacheSize:              500,
+			EnableEthRPC:             true,
```

Comment:
@rvagg @BigLep @rjan90 

Please see https://filecoinproject.slack.com/archives/CP50PPW2X/p1726327482392919

Reply:
These changes will be reverted as the final outcome of that discussion.